App ServerSocket/src/main/java/org/example/ClientHandler.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.time.LocalDateTime;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ClientHandler {
+    private static final Logger LOGGER = Logger.getLogger(ClientHandler.class.getName());
+    private Socket clientSocket;
+    private BufferedReader input;
+    private PrintWriter output;
+    private String regexPattern = ".*[а-яА-Я].*";
+
+    public ClientHandler(Socket clientSocket) throws IOException {
+        this.clientSocket = clientSocket;
+        input = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
+        output = new PrintWriter(clientSocket.getOutputStream());
+    }
```

Comment:
it is not a good idea to have logic in the constructor. And also if the constructor is throwing the exception